cdap-app-fabric/src/main/java/io/cdap/cdap/app/store/preview/PreviewStore.java

```diff
@@ -52,4 +57,38 @@
    * @param applicationId the id of the preview for which the data to be removed
    */
   void remove(ApplicationId applicationId);
+
+
+  /**
+   * Save the program run id associated with the preview run
+   *
+   * @param programRunId the program run id to save
+   */
+  void putProgramRunId(ProgramRunId programRunId);
```

Comment:
The naming convention is set and get. Better call this `setProgramRunId`.

Reply:
fixed